### PR TITLE
stage and prod canary commits with gevent enabled

### PR DESCRIPTION
**Pull request by duallain**

### CODE REVIEW

**Comment by pmac** [comment]
This looks great to me. My one question is whether we should update the `update_config.sh` script in bedrock to keep these deployment files updated with the latest image or not? If we want to manually manage the image for these I guess not, but we wouldn't want them to fall behind too far either.

File: iowa-a/bedrock-prod/canary-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+            secretKeyRef:
+              key: twitter-consumer-key
+              name: bedrock-prod-secrets
+        - name: TWITTER_CONSUMER_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: twitter-consumer-secret
+              name: bedrock-prod-secrets
+        - name: WEB_CONCURRENCY
+          value: "6"
```
Comment (sciurus) [inline iowa-a/bedrock-prod/canary-deploy.yaml:198]: This means we run 6 gevent workers per pod, which is the same number we did for meinheld?

File: iowa-a/bedrock-prod/canary-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+          value: "40dd9c0253db092c"
+        - name: FXA_OAUTH_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: fxa-oauth-client-secret
+              name: bedrock-prod-secrets
+        - name: GTM_CONTAINER_ID
+          value: GTM-MW3R8V
+        - name: GUNICORN_WORKER_CLASS
+          value: gevent
```
Comment (sciurus) [inline iowa-a/bedrock-prod/canary-deploy.yaml:123]: 👍

File: iowa-a/bedrock-prod/canary-deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+        - name: L10N_CRON
+          value: "True"
+        - name: LOGLEVEL
+          value: info
+        - name: MOFO_SECURITY_ADVISORIES_PATH
+          value: /tmp/mofo_security_advisories
+        - name: NEWRELIC_PYTHON_INI_FILE
+          value: newrelic.ini
+        - name: NEW_RELIC_APP_NAME
+          value: bedrock-prod-canary-iowa-a;bedrock-prod-canary
```
[the remark this answers is not in the document]
Comment (duallain) [reply iowa-a/bedrock-prod/canary-deploy.yaml:135]: Yes it should, I copied it from the prod yaml which has an explicit 'valueFrom' section linking to the configmap.  Taking that out and switching to straight value will take precedence.

File: iowa-a/bedrock-prod/canary-deploy.yaml (hunk 1 above)
Comment (duallain) [reply iowa-a/bedrock-prod/canary-deploy.yaml:198]: yes, same as meinheld.  Figured starting here with the canary made sense.

**Comment by duallain** [comment]
@pmac we can.  I tried to follow the naming pattern to make that easy, but wasn't sure if we wanted to keep these around for long enough that modifying the script would be worth it.  I suppose the script currently handles files not being in a particular folder, so it'd be safe to add canary.

**Review by sciurus — APPROVED** [review]

**Comment by pmac** [comment]
The script currently only looks for `deploy.yaml` or `clock-deploy.yaml`, but it wouldn't be hard to add these if we wanted. I can easily knock that out if it's something you think would be beneficial, even if these files aren't around that often.

**Comment by duallain** [comment]
@pmac https://github.com/mozilla/bedrock/pull/10035

**Pull request merged**